man/systemd.exec.xml

```diff
@@ -1187,7 +1187,76 @@
         <function>read</function> and <function>write</function>, and
         right after it add a blacklisting of
         <function>write</function>, then <function>write</function>
-        will be removed from the set.) </para></listitem>
+        will be removed from the set.)</para>
+
+        <para>As the number of possible system
+        calls is large, predefined sets of system calls are provided.
+        A set starts with <literal>@</literal> character, followed by
+        name of the set.
+
+        <table>
+          <title>Currently predefined system call sets</title>
+
+          <tgroup cols='2'>
+            <colspec colname='set' />
+            <colspec colname='description' />
+            <thead>
+              <row>
+                <entry>Set</entry>
+                <entry>Description</entry>
+              </row>
+            </thead>
+            <tbody>
+              <row>
+                <entry>@clock</entry>
+                <entry>System clock setting</entry>
+              </row>
+              <row>
+                <entry>@default</entry>
+                <entry>Above mentioned default list</entry>
+              </row>
+              <row>
+                <entry>@event-loop</entry>
+                <entry>Event loop use (poll/select)</entry>
```

Comment:
Also, I think we should give more hints what this includes, and use the usual C function syntax, i.e. something like this:

```
(<function>poll()</function>, <function>select()</function>, epoll(7), <function>eventfd</function>, …)
```

And the epoll reference should probably become a proper man page link